features/content_push/send_content_with_custom_media_fields.feature

```diff
@@ -0,0 +1,246 @@
+@content_push
+Feature: Handling the custom media fields
+  In order to be able to display galleries inside the article body
+  As a HTTP Client
+  I want to able to receive and parse the request with custom media fields payload
+
+  Scenario: Saving the data from custom media fields
+    Given I am authenticated as "test.user"
+    When I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/api/v1/assets/push" with parameters:
+      | key          | value                 |
+      | media_id     | 1234567890987654321a  |
+      | media        | @image.jpg            |
+    Then the response status code should be 201
+
+    And I am authenticated as "test.user"
```

Comment:
You don't need to be authenticted to use this endpoint. Instead that you can add header with security hash